core/templates/components/forms/schema-based-editors/schema-based-list-editor.component.html

```diff
@@ -13,8 +13,11 @@
     opacity: 1;
   }
   .schema-based-list-editor .oppia-new-tab-button-container {
-    height: 30px;
-    margin-top: 4px;
+    height: auto;
+    margin: 4px 0;
+  }
+  .schema-based-list-editor .oppia-new-tab-button-container .oppia-btn-no-display{
```

Comment:
```suggestion
  .schema-based-list-editor .oppia-new-tab-button-container .oppia-btn-no-display {
```

Reply:
@Rijuta-s i have added the bootstrap class. thanks!